packages/core/src/components/AlertBanner/__stories__/AlertBanner.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import AlertBanner from "../AlertBanner";
+import AlertBannerText from "../AlertBannerText/AlertBannerText";
+import AlertBannerLink from "../AlertBannerLink/AlertBannerLink";
+import AlertBannerButton from "../AlertBannerButton/AlertBannerButton";
+import { createStoryMetaSettingsDecorator } from "../../../storybook/functions/createStoryMetaSettingsDecorator";
+import "./AlertBanner.stories.scss";
+
+const metaSettings = createStoryMetaSettingsDecorator({
+  component: AlertBanner,
+  enumPropNamesArray: ["backgroundColor"]
+});
+
+interface Args {
+  bannerText: string;
+  linkText: string;
+  [props: string]: string;
+}
+
+const alertBannerTemplate = (args: Args) => {
+  return (
+    <AlertBanner {...args}>
+      <AlertBannerText text={args.bannerText} />
+      <AlertBannerLink text={args.linkText} href="https://monday.com" />
+    </AlertBanner>
+  );
+};
+
+export default {
+  title: "Feedback/AlertBanner",
+  component: AlertBanner,
+  argTypes: metaSettings.argTypes,
+  decorators: metaSettings.decorators
+};
+
+export const Overview = {
+  render: alertBannerTemplate.bind({}),
+  name: "Overview",
+
+  args: {
+    bannerText: "Alert banner message",
+    linkText: "this is a CTA",
+    className: "monday-storybook-alert-banner_big-container"
+  }
+};
+
+export const Types = {
```

Comment:
After adding the `Story` type, please make sure to type all the stories in the following way:
```suggestion
export const Types: Story = {
```